tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java

```diff
@@ -412,6 +412,27 @@ public void testGetAddressNames() throws Exception {
    }
 
    @Test
+   public void testGetAddressDeletedFromJournal() throws Exception {
```

Comment:
This test is passing for me even without the change in PostOfficeJournalLoader.java.

I wanted to figure out why I was not able to reproduce this with JUnit test the day before yesterday and I noticed what I said in the first paragraph. Do you have any ideas what might be wrong?